rust/src/fbs.rs

```diff
@@ -0,0 +1,26 @@
+//! Traits for converting between Rust and FlatBuffers data structures.
+
+pub(crate) trait TryFromFbs<'a>: Sized {
+    type FbsType;
+    type Error;
+
+    fn try_from_fbs(fbs: Self::FbsType) -> Result<Self, Self::Error>;
+}
+
+pub(crate) trait FromFbs: Sized {
```

Comment:
This traits take the argument via value or reference. There is no strong reason for each as both have their pros and cons. Any consideration here @nazar-pc ?

Reply:
I'd say use a reference if you can get away with it and do not reuse allocation anyway

Reply:
Done